adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -315,7 +316,10 @@ protected virtual Task PreTokenRequest()
 
         protected virtual async Task<AdalResultWrapper> SendTokenRequestAsync()
         {
-            var requestParameters = new DictionaryRequestParameters(this.Resource, this.ClientKey);
+            var requestParameters = new DictionaryRequestParameters(this.Resource, this.ClientKey)
+            {
+                { OAuth2Parameter.ClientInfo, "1" }
```

Comment:
>{ OAuth2Parameter.ClientInfo, "1" } [](start = 16, length = 35)

Why is this not added in AddAdditionalRequestParameters?

Reply:
we want to request it for all flows like in Msal, in AddAdditionalRequestParameters we add flow specific parameters

Reply:
the name of the method doesn't indicate that to me. Also is this really the only special parameter? As far as I can see you will get the same if you move this line to the Additional method. And for other places including the additionalparams they will also include this request param.

---
In reply to: [191035079](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1065#discussion_r191035079) [](ancestors = 191035079)

Reply:
it is not only special parameter , requestParameters is initialized with resourcem, clientId

Reply:
so this params are common for all flows, and AddAditionalRequestParameters is used to set flow specific params